extensions/data-transfer/portability-data-transfer-imgur/src/main/java/org/datatransferproject/datatransfer/imgur/photos/ImgurPhotosExporter.java

```diff
@@ -77,6 +82,27 @@ public ImgurPhotosExporter(
     this.objectMapper = objectMapper;
     this.monitor = monitor;
     this.jobStore = jobStore;
+    this.urlFactory = (urlString)->{return new URL(urlString);};
```

Comment:
I _think_ you can simplify this lambda a tiny bit

(and then also the diff I'm showing is assuming you go with the other comment i'm leaving you above)

```suggestion
    (url)-> new URL(url),
```

Reply:
changed, thanks!